.github/workflows/check.yml

```diff
@@ -3,15 +3,63 @@ name: Check
 on: [push, pull_request]
 
 jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+
+      - name: build and push
+        uses: docker/build-push-action@v6
+        with:
+          tags: django-pg-zero-downtime-migrations:latest
+          outputs: type=docker,dest=${{ runner.temp }}/myimage.tar
```

Comment:
can we provide more meaningful name for image, eg. `django-pg-zero-downtime-migrations-image`